test/iteration.jl

```diff
@@ -9,6 +9,7 @@ module TestIteration
 
     @test size(eachrow(df)) == (size(df, 1),)
     @test eachrow(df)[1] == DataFrameRow(df, 1)
+    @test typeof(collect(eachrow(df))) == Vector{DataFrameRow{DataFrame}}
```

Comment:
AFAIK this test should pass even without the change, right? Anyway it would be nice to test `eltype` too. BTW, you can use `isa` for a more compact code.

Reply:
No it would fail. Both `eachrow` and `eachcol` now produce `Vector{Any}`. I will correct the style.

Reply:
I have also corrected one more line which used `==` for testing type and I was following that style.